evm_arithmetization/tests/add11_yml.rs

```diff
@@ -266,8 +266,58 @@ fn add11_segments_aggreg() -> anyhow::Result<()> {
     )?;
     all_circuits.verify_segment_aggregation(&second_aggreg_proof)?;
 
-    let (txn_aggreg_proof, _) =
-        all_circuits.prove_transaction_aggregation(None, &second_aggreg_proof, second_aggreg_pv)?;
+    // We need two transactions to carry out a transaction aggregation. So we create
+    // a dummy segment aggregation.
```

Comment:
I know I've said it already, but I really don't see the point of adding yet another test that will be ignored by the CI, and hence never practically run, prone to be forgotten and broken with latest implementations. We already have
- empty_txn_proof which handles dummy cases
- log_opcode aggreg test which handles 2 non-dummy txns, then aggregated with a dummy block.

This seems more than enough, and already handling what's being reproduced here.